Development

300ms opened the pull request:
Microverse Project 4: Building with backgrounds and gradients

300ms committed:
let's go ^^

300ms committed:
html&body resized, a nav-bar added and building up its content

300ms committed:
nav-bar completed

300ms committed:
almost done, only last touches left now ^^

300ms committed:
resizings and html validation has been done ^^

300ms committed:
Let's Stickler ^^

300ms committed:
Fixing stickler errors ^^

300ms committed:
a few touches have been made lastly, and readme file is prepared ^^

300ms committed:
a few touches have been made lastly, and readme file is prepared ^^

300ms committed:
a few touches have been made lastly, and readme file is prepared ^^

300ms committed:
a few touches have been made lastly, and readme file is prepared ^^

samgaco commented on style.css:
```diff
@@ -0,0 +1,342 @@
+    -o-background-size: cover;
+    background-size: cover;
+}
+
```
**[REQUIRED]** The navigation bar should use a [`background -> gradient`](https://www.w3schools.com/cssref/css3_pr_background.asp) for background

samgaco commented on index.html:
```diff
@@ -0,0 +1,142 @@
+                <p class="footer-p">Copyright © 2014 Apple Inc. All rights reserved. <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/_1">Terms of Use</a> | <a href="https://web.archive.org/web/20140301004610/http://www.apple.com/legal/privacy/_1">Privacy Policy</a> </p>
+            </div>
+        </footer>
+    </main>
```
> The `<main>` element is used to denote the content of a webpage that relates to the central topic of that page or application. It should include content that is unique to that page and should not include content that is duplicated across multiple webpages, such as headers, footers, and primary navigation elements.

[Read more](https://html.com/tags/main/#ixzz5yaUVSP20)

300ms replied:
Yes you are right, it was my fault, i was careless a little bit but thanks for comments. 
I've fixed both problems now ^^

samgaco reviewed (changes requested):
#### Hi @300ms :high_brightness: 

#### Good job :+1: Everything looks good and the readme file is descriptive. There are some things to work on before we can give you the approval:

1. **[REQUIRED]** The navigation bar should use a [`background -> gradient`](https://www.w3schools.com/cssref/css3_pr_background.asp) for background.
2. **[REQUIERED]** You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.
3. Correct the usage of the `<main>` tag.

### So, please just work on these 3 :crossed_fingers: points and submit back for another code review. Regards!
---------

* **Friendly reminder:** You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. However, changing it now would take too much time. Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 
* I have seen your commits are a bit repetitive at some points, try to work on that aspect moving forward. More information [here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/).
* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

300ms committed:
a few touches have been made lastly, and readme file is prepared ^^

300ms committed:
a few touches have been made lastly, and readme file is prepared ^^

300ms committed:
background styling changed, images replaced with gradients

300ms committed:
background styling changed, images replaced with gradients

samgaco commented on index.html:
```diff
@@ -0,0 +1,142 @@
+
+<head>
+    <title>Apple</title>
+    <link rel="stylesheet" href="style.css">
```
**[REQUIERED]** As the [ validator ](https://validator.w3.org) is telling, The[ character encoding](https://www.w3schools.com/html/html_charset.asp) was not declared.

300ms replied:
@samgaco ,
You are right and thanks again ^^ Remembering all details, sometimes, is really hard ^^ Trying to get used to it with time. But as far as you show me my mistakes or suggesting better ways, i will try to follow. I will try to fix all you mentioned above before submitting project of course.

300ms replied:
@samgaco,
Also i should say, 

> 
> 
> **[REQUIERED]** As the [ validator ](https://validator.w3.org) is telling, The[ character encoding](https://www.w3schools.com/html/html_charset.asp) was not declared.

I validate my html code in same website, but i do it by "Direct Input". I select all code(ctrl+A) then paste it as input and check validation. I followed same path to validate and if you try that same way, you can see that error does not appear or even highlighted at all. I am not sure why but reason of not correcting that error is, did not see at all ^^

300ms replied:
Now it is all fixed except repetitive commits. Sorry about that really, i was working on some little details of/or same section/part/element/attribute or etc. and those moments are when i use same commits. I will try to write more clear and specific commits any more ^^

samgaco reviewed (approved):
#### Hi, again @300ms :high_brightness: 

#### Everything has been addressed except:

> **[REQUIERED]** You should fix the errors given to you by the[ validator ](https://validator.w3.org)when uploading the index file.

#### I approve the project since it is a very small change, but please :pray: correct that before merging.

##### You may merge to the master branch and submit the project completion form after addressing the requested change.

---------
### Read through these requirements for future projects, these are important :arrow_down: 
* Friendly reminder: You should use [Github flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in this project. However, changing it now would take too much time. Please make sure you use [Github Flow](https://microverse.pathwright.com/library/microverse-onboarding/86777/path/step/53987922/) in the next project. 
* **[For your next code review requests]**  Use meaningful names for your Pull Requests
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description
* I have seen your commits are a bit repetitive at some points, try to work on that aspect moving forward. More information [here](https://www.freecodecamp.org/news/level-you-up-to-awesome-commit-messages-a85558cb90e8/).

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

300ms committed:
Fixed html validation error for stylesheet(added "type=text/css")

Ispirett reviewed (approved):
Great Job this project is approved as @samgaco said :1st_place_medal: 

 Don't forget to add    ` <meta charset="UTF-8" /> to your head tag`

**Also Reminder**
 You should use Github flow in this project. However, changing it now would take too much time. Please make sure you use Github Flow in the next project.

300ms committed:
meta charset added under title tag ^^

300ms commented:
@Ispirett 
Hey ^^ thanks, sorry i fixed the "type=text/css" for importing it and looking at github flow again. Forgot the meta tag really but it is also fixed now.
